PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -384,7 +384,7 @@ bool ProjectPanel::saveWorkSpace()
 	} 
 }
 
-bool ProjectPanel::writeWorkSpace(const TCHAR *projectFileName)
+bool ProjectPanel::writeWorkSpace(const TCHAR *projectFileName, bool updateName)

```

Comment:
Please use `doUpdateGUI` instead of "updateName".

Reply:
Made the change as suggested.